blog/2015-02-26-Getting-Started-With_Docker-And-Wildfly.adoc

```diff
@@ -0,0 +1,195 @@
+= Getting started with Docker and Wildfly
+:page-layout: blog
+:page-author: rstryker
+:page-tags: [jbosstools, devstudio, server, docker, wildfly]
+
+So you've heard the buzzwords and all the hype around link:http://www.docker.com[Docker],
+but you haven't had the chance to play around with it yet or see if it fits your needs. 
+You're already an avid user of JBossTools, and are fairly familiar with starting and stopping
+your local or remote Wildfly installation, and deploying web applications to it. 
+
+If this describes you, and you're interested in trying out Docker, then this blog is targeted to you. 
+
+== Docker Installation
+
+Docker's installation method differs based on your platform. You'll want to 
+review link:https://docs.docker.com/installation/#installation[Docker's Installation] guides
+for your respective platforms. 
+
+== Running the default wildfly image
+
+Wildfly has published their docker images for public consumption. These images are very standard, 
+but a little restrictive for our needs. But, just to make sure your docker installation has worked, 
+and that Wildfly can start without any errors, we can do the following:
+
+ docker run -it jboss/wildfly
+
+This command will run the Docker image for jboss/wildfly in its default state with no customizations. 
+Once this is run, the command will not only start up your container, but also launch the server
+in standalone mode, and connect a terminal to it so you can see the output. 
+
+Assuming you see the console output, and the server runs with no errors, then you're 
+generally set up and ready for the next step. 
+
+To terminate the container, you can simply press `Ctrl+C`. If, for some reason, 
+your image has frozen, and `Ctrl+C` isn't working, you can also run the following
+commands to kill it forcefully. 
+
+Let's see what containers are currently running:
+
+....
+[root@localhost jbds_wf_external]$ docker ps
+CONTAINER ID        IMAGE                  COMMAND                CREATED             STATUS              PORTS                NAMES
+a798d07859f8        jboss/wildfly:latest   "/opt/jboss/wildfly/   4 seconds ago       Up 3 seconds        8080/tcp, 9990/tcp   backstabbing_poitras   
+....
+
+To kill the container, we'd then execute `docker kill a798d07859f8`, replacing the given hash with your CONTAINER ID. 
+
+
+== What's Missing
+
+The default docker image is lacking a few things for our needs. 
```

Comment:
After reading the whole thing - this paragraph feels out of place. Would be better to leave "what is missing" to the end *after* having shown what actually does work more or less out of the box.